src/docs/user/references/SchedulerGraphQLAPI.adoc

```diff
@@ -0,0 +1,380 @@
+
+====  Scheduler GraphQL API portal
+
+ProActive Scheduler GraphQL API portal can be accessed in your environment using the following link:
+
+++++
+<a class="schedulerGraphQLUrl" href="/scheduling-api/v1/graphiql" target="_blank">Scheduler GraphQL API</a> 
+++++
+
+In general the GraphiQL interface can be reached with `http://your-server:8080/scheduling-api/v1/graphiql`
+
+In order to use it, you must be first connected to one of the ProActive portals. If you are not currently connected or if your session has expired, you will receive a `MissingSessionIdException` when trying to run queries.
```

Comment:
To use it, you must be first connected to one of the ProActive portals. Otherwise, you will receive a `MissingSessionIdException` when trying to run queries.